gateway-developer-plugin/src/main/java/com/ca/apim/gateway/cagatewayconfig/CAGatewayDeveloper.java

```diff
@@ -102,7 +102,7 @@ private static BuildEnvironmentBundleTask createBuildEnvironmentBundleTask(@NotN
         // Create build-environment-bundle task
         final BuildEnvironmentBundleTask buildEnvironmentBundleTask = project.getTasks().create(BUILD_ENVIRONMENT_BUNDLE, BuildEnvironmentBundleTask.class, t -> {
```

Comment:
@uppoju can we remove the dependency task buildDeploymentBundleTask?

Reply:
OK